### PR TITLE
Changed title + logic of AccidentCountByDriverTypeWidget

### PR DESCRIPTION
**Pull request by shaysw**
- Changed title to not include section name
- Added filtering to include only drivers

### CODE REVIEW

File: anyway/widgets/suburban_widgets/accident_count_by_driver_type_widget.py [hunk 1]
```diff
@@ -29,9 +29,11 @@ def generate_items(self) -> None:
 
     @staticmethod
     def count_accidents_by_driver_type(request_params):
+        filters = get_injured_filters(request_params.location_info)
+        filters["involved_type"] = [1,2]
         involved_by_vehicle_type_data = get_accidents_stats(
             table_obj=InvolvedMarkerView,
-            filters=get_injured_filters(request_params.location_info),
+            filters=filters,
```
Comment (ziv17) [inline anyway/widgets/suburban_widgets/accident_count_by_driver_type_widget.py:36]: It looks like the filter is only on dates (and not for location).

File: anyway/widgets/suburban_widgets/accident_count_by_driver_type_widget.py [hunk 2]
```diff
@@ -29,9 +29,11 @@ def generate_items(self) -> None:
 
     @staticmethod
     def count_accidents_by_driver_type(request_params):
+        filters = get_injured_filters(request_params.location_info)
+        filters["involved_type"] = [1,2]
```
Comment (atalyaalon) [inline anyway/widgets/suburban_widgets/accident_count_by_driver_type_widget.py:33]: looks good, perhaps we should add it to our constants (backend_constants.py file) ?

**Review by atalyaalon — CHANGES_REQUESTED** [review]
@shaysw looks good! Thanks!
Added 2 comments.

**Comment by atalyaalon** [comment]
title modification made here: https://github.com/hasadna/anyway/pull/2171

**Comment by atalyaalon** [comment]
title modification made here: https://github.com/hasadna/anyway/pull/2171

**Comment by atalyaalon** [comment]
Closing this one, solved in https://github.com/hasadna/anyway/pull/2189/files

**Pull request closed without merging**